enterprise/templates/enterprise/enterprise_course_enrollment_error_page.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'enterprise/base.html' %}
+
+{% load staticfiles enterprise %}
+
+{% block contents %}
+  <div class="enterprise-container">
+    <div class="row">
+      <div class="col-3">
+        {% include "enterprise_course_enrollment_sidebar_column.html" %}
```

Comment:
Can you name this partial `_enterprise_customer_sidebar.html`?

Reply:
Sure.

Reply:
Done.